document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/service/impl/AccountsServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.document.generator.accounts.service.impl;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.accounts.Accounts;
+import uk.gov.companieshouse.document.generator.accounts.data.accounts.AccountsManager;
+import uk.gov.companieshouse.document.generator.accounts.service.AccountsService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Service
+public class AccountsServiceImpl implements AccountsService {
+
+    private static final Logger LOG = LoggerFactory.getLogger("document-generator-accounts");
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Accounts getAccounts(String resource) {
+        LOG.info("Getting accounts data from abridged-accounts-api");
```

Comment:
Is it worth including the `resource` in the log so it's more meaningful to support?

Reply:
As mentioned in my other comment, logging will be fine-tuned in a separate PR :)

Reply:
Logging should contain all relevant information. This could be missed in a future PR.

Reply:
It won't be missed purely because I've already done the work and I am simply copy and pasting small chunk at a time just to keep the PRs small.

Regarding the second point, the reason why `abridged-api` is mentioned is because I want there to be a clear separation between the way we use the `company-accounts.api` and the existing `abridged-api`. We already know that it will eventually get migrated to the new api, but for the time being I have made a clear distinction because it will make it much easier to take out in future and makes it easier to debug in regards to knowing where the error comes from.